README.md

```diff
@@ -145,38 +145,86 @@ $http.get('https://adobeioruntime.net/api/v1/web/helix/helix-services/status@v3/
 );
 ```
 
-## Automated Update of Synthetics Checks
+## Automated Monitoring
 
-`helix-status` provides a small command line tool called `synthetics-check` that enables the automated update or creation of checks in New Relics Synthetics. It is intended to be run as part of your deployment pipeline and allows you to keep the health check format and the extracted custom fields in sync.
+`helix-status` provides the following command line tools intended to be run as part of your deployment pipeline to automate monitoring:
+
+### Statuspage: Automated Update of Components
+
+`statuspage` allows to automatically create components in Statuspage.
+
+Usage:
+
+```bash
+$ npx statuspage
+statuspage <cmd>
+
+Commands:
+  statuspage create  Create a new component
+
+
+Options:
+  --auth      The Statuspage API key (required)                        [string]
+  --page_id   The ID of the page to add the component to (required)    [string]
+  --group_id  The ID of the component group (optional)                 [string]
+  --version   Show version number                                      [boolean]
+  --help      Show help                                                [boolean]
+
+$ npx statuspage create --page_id=zk3f17cq2qhw --group_id=qkf4y9ghhlgw
+Creating a new component @adobe/helix-status in group Foo Services
+Automation email: component+abcdef01-2345-6789-abcd-ef0123456789@notifications.statuspage.io
+done.
+```
+
+By default, the check will use the `name` from your `package.json`, but you can override it using the `--name` parameter.
+
+`statuspage` requires a Statuspage [API Key](https://developer.statuspage.io/#section/Authentication) that should be passed using either the `--auth` parameter or the `STATUSPAGE_AUTH` environment variable, as well as a Statuspage [Page ID] that should be passed using either the `--page_id` parameter or the `STATUSPAGE_PAGE_ID` environment variable. 
+
+
+### New Relic: Automated Update of Synthetics Checks, Alert Policies and Notification Channels
+
+`newrelic` automates the following New Relic features:
+1. creation or update of monitors in New Relics Synthetics
+1. creation of notification channels in New Relic Alerts
+1. creation or update of alert policies in New Relic Alerts
+1. linking of alert policies to monitors and notification channels
 
 Usage:
 
 ```bash
-$ npx synthetics-check
-synthetics-check <cmd> url
+$ npx newrelic
+newrelic-synthetics <cmd> url
```

Comment:
This looks inconsistent, it should return the name of the command entered above.

Reply:
Good catch. I changed tool names back and forth but eventually settled for simplified versions: `statuspage` and `newrelic`